Modules/Core/Common/src/itkObjectFactoryBase.cxx

```diff
@@ -122,7 +123,7 @@ class ObjectFactoryBase::ObjectFactoryBasePrivate : public LightObject
 
   FactoryListType m_RegisteredFactories{};
   FactoryListType m_InternalFactories{};
-  bool            m_Initialized{ false };
+  std::atomic<bool> m_Initialized{ false };
   bool            m_StrictVersionChecking{ false };
```

Comment:
I'm sorry (it wasn't my preference) but the ITK formatting rules require the names of consecutively declared member variables to be aligned (assuming a fixed-size font). So the surrounding member variables should have an extra space, in order to pass clang-format checking on the CI.

Reply:
Hmm, OK, done.

The downside of that style is that it pollutes history, and will make the results of git blame less useful. Also makes merge conflicts more likely. Oh well.

Reply:
> The downside of that style is that it pollutes history, and will make the results of git blame less useful. Also makes merge conflicts more likely. 

I agree, but for the time being, this is the ITK coding style 🤷 Maybe when we're going to upgrade the clang-format version, we might reconsider the formatting rules... I hope!

Reply:
I also dislike declaration alignment.